tests/src/tests/roact.spec.tsx

```diff
@@ -285,12 +285,6 @@ export = () => {
 		).to.equal(ElementKind.Host);
```

Comment:
It's probably a good idea to add tests for `JsxText`. Should be possible to do by just defining a component that takes string children?

Reply:
Yeah I'd like to make a simple "baseline JSX implementation" which just returns whatever you pass into it. This would be a better test of the compiler than the current roact tests.